backend/inspirehep/search/mappings/v7/records/hep.json

```diff
@@ -1497,6 +1498,11 @@
           "char_filter": ["dot_to_whitespace_normalizer"],
           "tokenizer": "standard",
           "type": "custom"
+        },
+        "page_artid_analyzer": {
+          "char_filter": ["remove_leading_zero_normalizer"],
+          "tokenizer": "standard",
```

Comment:
do we really want to go through the tokenizer? I think it would be better to keep it as a `keyword` but use a normalizer, otherwise strange things might happen, I don't exactly understand the rules for splitting words containing chars+digits which might occur here.

Reply:
I don't think will make any difference in that case, but I've updated it